vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -264,17 +265,35 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
     return this.loginSource === LoginSource.IDIR
   }
 
-  get isBCSC (): boolean {
-    return this.loginSource === LoginSource.BCSC
+  get isBceid (): boolean {
+    return this.loginSource === LoginSource.BCEID
+  }
+
+  get isBcscOrBceid (): boolean {
+    return [LoginSource.BCSC.valueOf(), LoginSource.BCEID.valueOf()].indexOf(this.loginSource) >= 0
   }
 
   private async mounted () {
     getModule(AccountModule, this.$store)
     getModule(AuthModule, this.$store)
     this.syncWithSessionStorage()
     if (this.isAuthenticated) {
-      this.loadUserInfo()
+      await this.loadUserInfo()
       await this.syncAccount()
+      await this.updateProfile()
+    }
+  }
+
+  @Watch('isAuthenticated')
+  private async onisAuthenticated (isAuthenitcated: string, oldVal: string) {
+    if (isAuthenitcated) {
+      await this.updateProfile()
+    }
+  }
+
+  private async updateProfile () {
+    if (this.isBceid) {
```

Comment:
Is this correct? isBceid is a function right ?

Reply:
a getter..so will work i guess